docs/setting-up/server/easy-install.md

```diff
@@ -4,10 +4,20 @@
     - This is a [technical preview] and is subject to change.
     - Download and check `get-pmm2.sh` before running it to make sure you know what it does.
 
+### Linux:
 ```sh
-curl -fsSL https://raw.githubusercontent.com/percona/pmm/PMM-2.0/get-pmm.sh -o get-pmm2.sh && \
-chmod +x get-pmm2.sh && \
-./get-pmm2.sh
+curl -fsSL -O https://raw.githubusercontent.com/percona/pmm/PMM-2.0/get-pmm.sh -O https://raw.githubusercontent.com/percona/pmm/PMM-2.0/.sha256-oneline && \
+sha256sum .sha256-oneline -c && \
```

Comment:
`sha256sum` is not installed by default on MacOS, while `shasum` is. Can we rewrite this to use `shasum`, which seem to be able to handle them all with a parameter `-a ...`?

Reply:
We also don't have `shasum` on Linux (CentOs) by default :(
It's part of `perl-Digest-SHA` package.

I think I can run different tools depends on OS but the script will be more complicated.

Reply:
Ok then, let's leave it as is )